packages/myst-transforms/src/links/types.ts

```diff
@@ -5,6 +5,7 @@ import type { Inventory } from 'intersphinx';
 
 export interface LinkTransformer {
   protocol?: string;
+  formatsText?: boolean;
```

Comment:
This is not set to true anywhere?

Reply:
It should be opt-out (explicit `false`). Looks like I missed it in `wiki.ts`. Let's fix that.